testing/stgit/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Joseph Benden <joe@benden.us>
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=stgit
+pkgver=0.19
+pkgrel=0
+pkgdesc="Manage a stack of patches using GIT as a backend"
+arch="noarch"
+url="http://www.procode.org/stgit/"
+license="GPL"
+options="!check !strip" # unit-tests fail due to iconv usage
+depends="git python3"
+makedepends="python3 asciidoc xmlto perl"
```

Comment:
python3 is in depends=, doesn't need to be in makedepends

Reply:
Fixed. Thanks!